slack/plugin.py

```diff
@@ -17,24 +17,38 @@ class SlackUser:
     display_name: str
 
 
-def _with_mentions(
-    users: list[SlackUser] | Callable[[], list[SlackUser]],
+async def _with_mentions(
+    users: Union[
+        list[SlackUser],
+        Callable[[], list[SlackUser]],
+        Callable[[], Coroutine[Any, Any, list[SlackUser]]],
+    ],
```

Comment:
```suggestion
async def _with_mentions(
    users: Union[list[SlackUser], Awaitable[list[SlackUser]]],
```
I think the `Callable[[], list[SlackUser]]` is now unused, and I think it is simpler for the current use to use the coroutine as the abstraction for deferred computation, since it seems likely that the use will continue to be like that for the future I can foresee.

Reply:
Done.